src/bindings/python/tests/test_runtime/test_core.py

```diff
@@ -273,27 +271,15 @@ def test_register_plugins(device):
     full_device_name = core.get_property(device, "FULL_DEVICE_NAME")
     plugins_xml = plugins_path(device, full_device_name)
     core.register_plugins(plugins_xml)
-    model = core.read_model(model=test_net_xml, weights=test_net_bin)
-    compiled_model = core.compile_model(model, "CUSTOM")
-    os.remove(plugins_xml)
```

Comment:
```suggestion
os.remove(plugins_xml)
```
keep this line as it cleaned produced file plugins_xml

Reply:
Thanks! Reverted